patches/node.v22.8.0.cpp.patch

```diff
@@ -0,0 +1,686 @@
+diff --git node/common.gypi node/common.gypi
+index a97e77860e..09444e795f 100644
+--- node/common.gypi
++++ node/common.gypi
+@@ -187,7 +187,7 @@
+             ['clang==1', {
+               'lto': ' -flto ', # Clang
+             }, {
+-              'lto': ' -flto=4 -fuse-linker-plugin -ffat-lto-objects ', # GCC
++              'lto': ' -flto=4 -ffat-lto-objects ', # GCC
+             }],
+           ],
+         },
+diff --git node/deps/ngtcp2/nghttp3/lib/nghttp3_ringbuf.c node/deps/ngtcp2/nghttp3/lib/nghttp3_ringbuf.c
+index 61a7d06cad..eeebf67796 100644
+--- node/deps/ngtcp2/nghttp3/lib/nghttp3_ringbuf.c
++++ node/deps/ngtcp2/nghttp3/lib/nghttp3_ringbuf.c
+@@ -33,17 +33,6 @@
+ 
+ #include "nghttp3_macro.h"
+ 
+-#if defined(_MSC_VER) && !defined(__clang__) &&                                \
+-    (defined(_M_ARM) || defined(_M_ARM64))
+-unsigned int __popcnt(unsigned int x) {
+-  unsigned int c = 0;
+-  for (; x; ++c) {
+-    x &= x - 1;
+-  }
+-  return c;
+-}
+-#endif
+-
```

Comment:
let me preface this by saying that I would not consider myself a C++ expert either :grimacing: 

this is an intrinsic function needed for <=MSVC2019 on ARM64 and is no longer needed here (causing a compile error because it is already defined) - it would be possible to add a version condition instead but it would result in patch changes either way..